azure-pipelines.yml

```diff
@@ -191,9 +191,9 @@ jobs:
       )
     commitMessage: dependencies['git_commit']['outputs']['commit.message']
     matrix:
-      py38_conda_forge_openblas_ubuntu_2204:
+      py39_conda_forge_openblas_ubuntu_2204:
```

Comment:
I think it'll be good to change the keys to be version independent. Maybe this:

```suggestion
      py_min_conda_forge_openblas_ubuntu_latest_lts:
```

This way we do not need to change the key every time we update the version.

Reply:
I thought about the same. Let me do that.

Reply:
I did it for all py39